exp/addons/controllers/suite_test.go

```diff
@@ -38,7 +38,7 @@ func TestMain(m *testing.M) {
 	fmt.Println("Creating new test environment")
 	testEnv = helpers.NewTestEnvironment()
 
-	trckr, err := remote.NewClusterCacheTracker(log.NullLogger{}, testEnv.Manager)
+	trckr, err := remote.NewClusterCacheTracker(testEnv.Manager, remote.SetLogger(log.NullLogger{}))
```

Comment:
```suggestion
	trckr, err := remote.NewClusterCacheTracker(testEnv.Manager)
```
Give it is already the default